report.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites></testsuites>
```

Comment:
Why did you add this file?

Reply:
Ugh shouldn't have gone in. Not sure if this is something we want to add to the `.gitignore`

Reply:
How did the file got generated?